TECHMANIA/Assets/Scripts/GameSetup.cs

```diff
@@ -13,7 +13,7 @@ public static string trackFolder
     {
         get
         {
-            return new FileInfo(trackPath).DirectoryName;
+            return new FileInfo(trackPath).DirectoryName.Replace("/jar:file:/data", "jar:file:///data");
```

Comment:
`// FileInfo.DirectoryName turns "jar:file:///" into "/jar:file:/", so we need to correct it.`